app/models/system_tag.rb

```diff
@@ -0,0 +1,35 @@
+class SystemTag < ApplicationRecord
```

Comment:
Similarly, should this be `OrganizationTag`?

Reply:
Nope, but maybe, like SystemTags, we could rename them to TagLookup, that might help?
They're going to populate the filter list on Share page, and should show as suggested tags underneath anything taggable.